packages/nocodb/src/modules/jobs/jobs.gateway.ts

```diff
@@ -14,13 +14,21 @@ import { JobEvents } from '../../interface/Jobs';
 import type { OnModuleInit } from '@nestjs/common';
 import type { JobStatus } from '../../interface/Jobs';
 
+const url = new URL(process.env.NC_PUBLIC_URL || `http://localhost:${process.env.PORT || '8080'}/`)
+let namespace = url.pathname
+if (!namespace.endsWith('/')) {
+  namespace = namespace + '/jobs';
+} else {
+  namespace = namespace + 'jobs';
+}
```

Comment:
```js
let namespace = url.pathname
namespace += namespace.endsWith("/") ? "jobs" : "/jobs"
```

Reply:
Modified it with the code you suggested.